src/compose.js

```diff
@@ -6,6 +6,9 @@ var reverse = require('./reverse');
  * Performs right-to-left function composition. The rightmost function may have
  * any arity; the remaining functions must be unary.
  *
+ * The result of compose is not automatically curried so functions with optional
+ * arguments can be used as the rightmost function and are called immediately.
+ *
```

Comment:
I'd suggest it be even more bare-bones.  As the discussion in #1391 showed, the biggest problem was not optional arguments (or at least not as we typically think of them) but with passing partially configured functions along to another function.

Perhaps just

    **Note**: the result of compose is not automatically curried.